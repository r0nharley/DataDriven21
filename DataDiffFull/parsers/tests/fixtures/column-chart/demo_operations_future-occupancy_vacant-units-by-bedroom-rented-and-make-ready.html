<div class="rl-chart  chart-column" ng-switch-when="chart" config="config" filters="filters" ng-class="{ 'rl-highlight': showHighlight, 'rl-no-header': config.noHeader, 'rl-viz-sm': vizSize == 'sm',  'rl-viz-md': vizSize == 'md', 'rl-viz-lg': vizSize == 'lg' }" viz-size="vizSize" id="chart8">
  <!-- ngIf: alertType -->
  <!-- ngIf: loading -->
  <!-- ngIf: !isEmpty -->
  <div class="rl-viz-header" ng-if="!isEmpty">
    <div class="rl-title rl-small-width" title="Vacant Units by Bedroom, Rented &amp; Make Ready Status" ng-class="{ 'rl-small-width': showGlobalFiltersTooltip }">Vacant Units by Bedroom, Rented &amp; Make Ready Status</div>
    <!-- ngIf: chart && chart.description -->
    <div class="fa fa-info rl-viz-header-icon rl-description-tooltip-icon" ng-if="chart &amp;&amp; chart.description" uib-tooltip-html="htmlDescriptionTooltip" tooltip-placement="auto bottom-left" tooltip-class="rl-viz-header-tooltip"></div>
    <!-- end ngIf: chart && chart.description -->
    <!-- ngIf: showGlobalFiltersTooltip -->
    <div class="fa fa-globe rl-viz-header-icon rl-global-filters-icon" ng-if="showGlobalFiltersTooltip" uib-tooltip="Affects Global Filters" tooltip-placement="auto bottom-left" tooltip-class="rl-viz-header-tooltip"></div>
    <!-- end ngIf: showGlobalFiltersTooltip -->
    <!-- ngIf: showDragHandle() -->
    <!-- ngIf: (chart && hasData) || (config().isPinboard && !loading) -->
    <div class="rl-viz-dropdown" rl-window-resize="resetDropDownPosition()" ng-if="(chart &amp;&amp; hasData) || (config().isPinboard &amp;&amp; !loading)" export-image="exportImage" viz="config().viz" dashboard-filters="filters()" options-list="optionsList" is-pinboard="config().isPinboard">
      <div class="rl-dropdown-menu btn-group rl-dropdown-viz-menu" ng-class="{ open: isOpen, 'rl-dropdown-header-menu': dropdownStyle() == 'header-menu', 'rl-dropdown-viz-menu': dropdownStyle() == 'viz-menu' }" dropdown-style="dropdownStyle" menu-alignment="menuAlignment" config="config" on-select="onOptionSelect">
        <button class="btn btn-info dropdown-toggle" ng-click="toggleMenu()">
					<span ng-show="dropdownStyle() == 'header-menu'" class="ng-hide">
						<span class="caret"></span>
					</span>
					<span ng-show="dropdownStyle() == 'viz-menu'">
						<span class="fa fa-bars"></span>
					</span>
        </button>
        <!-- ngIf: isOpen -->
        <ul class="dropdown-menu rl-right" ng-class="{ 'rl-right': menuAlignment() == 'right' }">
          <!-- ngRepeat: option in config().options -->
          <li ng-repeat="option in config().options">
            <!-- ngIf: option.key !== 'addToPinboard' -->
            <!-- ngIf: option.key === 'addToPinboard' -->
            <a ng-if="option.key === 'addToPinboard'" ng-class="{ 'rl-open': isPinboardSubmenuShown }" ng-mouseover="togglePinboardMenu(true)">
              <span class="pull-left">Pinboards</span>
              <span class="pull-right fa fa-thumb-tack fa-rotate-36" ng-class="option.classDefault"></span>
              <div class="rl-clear"></div>
            </a>
            <!-- end ngIf: option.key === 'addToPinboard' -->
          </li>
          <!-- end ngRepeat: option in config().options -->
          <li ng-repeat="option in config().options">
            <!-- ngIf: option.key !== 'addToPinboard' -->
            <a ng-if="option.key !== 'addToPinboard'" ng-click="onOptionClick(option)" ng-disabled="option.isLoading" ng-mouseover="togglePinboardMenu(false)">
              <span class="pull-left">Save as Image</span>
              <span class="pull-right fa fa-file-image-o" ng-class="option.isLoading? option.classLoading : option.classDefault"></span>
              <div class="rl-clear"></div>
            </a>
            <!-- end ngIf: option.key !== 'addToPinboard' -->
            <!-- ngIf: option.key === 'addToPinboard' -->
          </li>
          <!-- end ngRepeat: option in config().options -->
          <li ng-repeat="option in config().options">
            <!-- ngIf: option.key !== 'addToPinboard' -->
            <a ng-if="option.key !== 'addToPinboard'" ng-click="onOptionClick(option)" ng-disabled="option.isLoading" ng-mouseover="togglePinboardMenu(false)">
              <span class="pull-left">Export to CSV</span>
              <span class="pull-right fa fa-download" ng-class="option.isLoading? option.classLoading : option.classDefault"></span>
              <div class="rl-clear"></div>
            </a>
            <!-- end ngIf: option.key !== 'addToPinboard' -->
            <!-- ngIf: option.key === 'addToPinboard' -->
          </li>
          <!-- end ngRepeat: option in config().options -->
        </ul>
        <div class="rl-viz-dropdown-pinboard-submenu ng-hide" rl-window-resize="realignMenu()" ng-show="isPinboardSubmenuShown" viz="config().viz" visible="isPinboardSubmenuShown">
          <ul>
            <!-- ngRepeat: option in pinboardOptions -->
          </ul>
        </div>
      </div>
    </div>
    <!-- end ngIf: (chart && hasData) || (config().isPinboard && !loading) -->
  </div>
  <!-- end ngIf: !isEmpty -->
  <!-- ngIf: !hasData && !loading && !alertType && !isEmpty -->
  <!-- ngIf: isEmpty -->
  <!-- ngIf: !isEmpty -->
  <div class="rl-chart-content" ng-if="!isEmpty" data-highcharts-chart="6">
    <div id="highcharts-nr9wxib-19" class="highcharts-container " style="position: relative; overflow: hidden; width: 405px; height: 427px; text-align: left; line-height: normal; z-index: 0; -webkit-tap-highlight-color: rgba(0, 0, 0, 0);">
      <svg version="1.1" class="highcharts-root" style="font-family:&quot;Lucida Grande&quot;, &quot;Lucida Sans Unicode&quot;, Arial, Helvetica, sans-serif;font-size:12px;"
           xmlns="http://www.w3.org/2000/svg" width="405" height="427">
        <desc>Created with Highstock 5.0.6</desc>
        <defs>
          <clipPath id="highcharts-nr9wxib-20">
            <rect x="0" y="0" width="348" height="263" fill="none"></rect>
          </clipPath>
          <clipPath id="highcharts-nr9wxib-21">
            <rect x="0" y="0" width="348" height="263" fill="none"></rect>
          </clipPath>
        </defs>
        <rect fill="#ffffff" class="highcharts-background" x="0" y="0" width="405" height="427" rx="0" ry="0"></rect>
        <rect fill="none" class="highcharts-plot-background" x="47" y="20" width="348" height="263"></rect>
        <g class="highcharts-grid highcharts-xaxis-grid ">
          <path fill="none" stroke="#D8D8D8" stroke-width="0" class="highcharts-grid-line" d="M 78.5 20 L 78.5 283" opacity="1"></path>
          <path fill="none" stroke="#D8D8D8" stroke-width="1" class="highcharts-grid-line" d="M 109.5 20 L 109.5 283" opacity="1"></path>
          <path fill="none" stroke="#D8D8D8" stroke-width="0" class="highcharts-grid-line" d="M 141.5 20 L 141.5 283" opacity="1"></path>
          <path fill="none" stroke="#D8D8D8" stroke-width="1" class="highcharts-grid-line" d="M 173.5 20 L 173.5 283" opacity="1"></path>
          <path fill="none" stroke="#D8D8D8" stroke-width="0" class="highcharts-grid-line" d="M 204.5 20 L 204.5 283" opacity="1"></path>
          <path fill="none" stroke="#D8D8D8" stroke-width="1" class="highcharts-grid-line" d="M 236.5 20 L 236.5 283" opacity="1"></path>
          <path fill="none" stroke="#D8D8D8" stroke-width="0" class="highcharts-grid-line" d="M 267.5 20 L 267.5 283" opacity="1"></path>
          <path fill="none" stroke="#D8D8D8" stroke-width="1" class="highcharts-grid-line" d="M 299.5 20 L 299.5 283" opacity="1"></path>
          <path fill="none" stroke="#D8D8D8" stroke-width="1" class="highcharts-grid-line" d="M 331.5 20 L 331.5 283" opacity="1"></path>
          <path fill="none" stroke="#D8D8D8" stroke-width="0" class="highcharts-grid-line" d="M 362.5 20 L 362.5 283" opacity="1"></path>
          <path fill="none" stroke="#D8D8D8" stroke-width="1" class="highcharts-grid-line" d="M 395.5 20 L 395.5 283" opacity="1"></path>
          <path fill="none" stroke="#D8D8D8" stroke-width="0" class="highcharts-grid-line" d="M 46.5 20 L 46.5 283" opacity="1"></path>
        </g>
        <g class="highcharts-grid highcharts-yaxis-grid ">
          <path fill="none" stroke="#D8D8D8" stroke-width="1" stroke-dasharray="1,3" class="highcharts-grid-line" d="M 47 283.5 L 395 283.5" opacity="1"></path>
          <path fill="none" stroke="#D8D8D8" stroke-width="1" stroke-dasharray="1,3" class="highcharts-grid-line" d="M 47 239.5 L 395 239.5" opacity="1"></path>
          <path fill="none" stroke="#D8D8D8" stroke-width="1" stroke-dasharray="1,3" class="highcharts-grid-line" d="M 47 195.5 L 395 195.5" opacity="1"></path>
          <path fill="none" stroke="#D8D8D8" stroke-width="1" stroke-dasharray="1,3" class="highcharts-grid-line" d="M 47 152.5 L 395 152.5" opacity="1"></path>
          <path fill="none" stroke="#D8D8D8" stroke-width="1" stroke-dasharray="1,3" class="highcharts-grid-line" d="M 47 108.5 L 395 108.5" opacity="1"></path>
          <path fill="none" stroke="#D8D8D8" stroke-width="1" stroke-dasharray="1,3" class="highcharts-grid-line" d="M 47 64.5 L 395 64.5" opacity="1"></path>
          <path fill="none" stroke="#D8D8D8" stroke-width="1" stroke-dasharray="1,3" class="highcharts-grid-line" d="M 47 19.5 L 395 19.5" opacity="1"></path>
        </g>
        <rect fill="none" class="highcharts-plot-border" x="47" y="20" width="348" height="263"></rect>
        <g class="highcharts-axis highcharts-xaxis ">
          <path fill="none" strokeWidth="0" d="M 47 283 L 47 366.4918435941019 M 79 283 L 79 343.4918435941019 M 110 343.4918435941019 L 110 366.4918435941019 M 110 283 L 110 343.4918435941019 M 110 343.4918435941019 L 110 366.4918435941019 M 142 283 L 142 343.4918435941019 M 174 343.4918435941019 L 174 366.4918435941019 M 174 283 L 174 343.4918435941019 M 174 343.4918435941019 L 174 366.4918435941019 M 205 283 L 205 343.4918435941019 M 237 343.4918435941019 L 237 366.4918435941019 M 237 283 L 237 343.4918435941019 M 237 343.4918435941019 L 237 366.4918435941019 M 268 283 L 268 343.4918435941019 M 300 343.4918435941019 L 300 366.4918435941019 M 300 283 L 300 343.4918435941019 M 300 343.4918435941019 L 300 366.4918435941019 M 332 283 L 332 343.4918435941019 M 332 343.4918435941019 L 332 366.4918435941019 M 363 283 L 363 343.4918435941019 M 396 343.4918435941019 L 396 366.4918435941019 M 396 283 L 396 343.4918435941019 M 396 343.4918435941019 L 396 366.4918435941019 M 47 343 L 395 343 M 47 366 L 395 366" visibility="visible"></path>
          <path fill="none" class="highcharts-axis-line" stroke="#ccd6eb" stroke-width="1" d="M 47 283.5 L 395 283.5"></path>
        </g>
        <g class="highcharts-axis highcharts-yaxis ">
          <path fill="none" class="highcharts-axis-line" d="M 47 20 L 47 283"></path>
        </g>
        <g class="highcharts-series-group">
          <g class="highcharts-series highcharts-series-0 highcharts-column-series highcharts-color-undefined highcharts-tracker " transform="translate(47,20) scale(1 1)" width="263" height="348" clip-path="url(https://staging.rentlytics.com/demo/bi/operations-future-occupancy/visualization#highcharts-nr9wxib-21)">
            <rect x="7.5" y="238.5" width="16" height="4" fill="#00cee6" class="highcharts-point" stroke="#ffffff" stroke-width="1"></rect>
            <rect x="39.5" y="237.5" width="16" height="5" fill="#00cee6" class="highcharts-point" stroke="#ffffff" stroke-width="1"></rect>
            <rect x="70.5" y="51.5" width="16" height="84" fill="#00cee6" class="highcharts-point " stroke="#ffffff" stroke-width="1"></rect>
            <rect x="102.5" y="142.5" width="16" height="39" fill="#00cee6" class="highcharts-point" stroke="#ffffff" stroke-width="1"></rect>
            <rect x="134.5" y="37.5" width="16" height="119" fill="#00cee6" class="highcharts-point " stroke="#ffffff" stroke-width="1"></rect>
            <rect x="165.5" y="107.5" width="16" height="72" fill="#00cee6" class="highcharts-point" stroke="#ffffff" stroke-width="1"></rect>
            <rect x="197.5" y="182.5" width="16" height="49" fill="#00cee6" class="highcharts-point" stroke="#ffffff" stroke-width="1"></rect>
            <rect x="229.5" y="233.5" width="16" height="11" fill="#00cee6" class="highcharts-point" stroke="#ffffff" stroke-width="1"></rect>
            <rect x="260.5" y="261.5" width="16" height="2" fill="#00cee6" class="highcharts-point" stroke="#ffffff" stroke-width="1"></rect>
            <rect x="292.5" y="251.5" width="16" height="3" fill="#00cee6" class="highcharts-point" stroke="#ffffff" stroke-width="1"></rect>
          </g>
          <g class="highcharts-markers highcharts-series-0 highcharts-column-series highcharts-color-undefined " transform="translate(47,20) scale(1 1)" width="263" height="348" clip-path="none"></g>
          <g class="highcharts-series highcharts-series-1 highcharts-column-series highcharts-color-undefined highcharts-tracker" transform="translate(47,20) scale(1 1)" width="263" height="348" clip-path="url(https://staging.rentlytics.com/demo/bi/operations-future-occupancy/visualization#highcharts-nr9wxib-21)">
            <rect x="7.5" y="242.5" width="16" height="21" fill="#218a8c" class="highcharts-point" stroke="#ffffff" stroke-width="1"></rect>
            <rect x="39.5" y="242.5" width="16" height="21" fill="#218a8c" class="highcharts-point" stroke="#ffffff" stroke-width="1"></rect>
            <rect x="70.5" y="135.5" width="16" height="128" fill="#218a8c" class="highcharts-point" stroke="#ffffff" stroke-width="1"></rect>
            <rect x="102.5" y="181.5" width="16" height="82" fill="#218a8c" class="highcharts-point" stroke="#ffffff" stroke-width="1"></rect>
            <rect x="134.5" y="156.5" width="16" height="107" fill="#218a8c" class="highcharts-point" stroke="#ffffff" stroke-width="1"></rect>
            <rect x="165.5" y="179.5" width="16" height="84" fill="#218a8c" class="highcharts-point" stroke="#ffffff" stroke-width="1"></rect>
            <rect x="197.5" y="231.5" width="16" height="32" fill="#218a8c" class="highcharts-point" stroke="#ffffff" stroke-width="1"></rect>
            <rect x="229.5" y="244.5" width="16" height="19" fill="#218a8c" class="highcharts-point" stroke="#ffffff" stroke-width="1"></rect>
            <rect x="292.5" y="254.5" width="16" height="9" fill="#218a8c" class="highcharts-point" stroke="#ffffff" stroke-width="1"></rect>
            <rect x="324.5" y="259.5" width="16" height="4" fill="#218a8c" class="highcharts-point" stroke="#ffffff" stroke-width="1"></rect>
          </g>
          <g class="highcharts-markers highcharts-series-1 highcharts-column-series highcharts-color-undefined " transform="translate(47,20) scale(1 1)" width="263" height="348" clip-path="none"></g>
        </g>
        <g class="highcharts-legend" transform="translate(114,386)">
          <rect fill="none" class="highcharts-legend-box" rx="0" ry="0" x="0" y="0" width="177" height="26" visibility="visible"></rect>
          <g>
            <g>
              <g class="highcharts-legend-item highcharts-column-series highcharts-color-undefined highcharts-series-0" transform="translate(8,3)">
                <text x="21" style="color:#333333;font-size:12px;font-weight:normal;cursor:pointer;fill:#333333;" text-anchor="start" y="15">Rented</text>
                <rect x="2" y="4" width="12" height="12" fill="#00cee6" rx="6" ry="6" class="highcharts-point"></rect>
              </g>
              <g class="highcharts-legend-item highcharts-column-series highcharts-color-undefined highcharts-series-1" transform="translate(91.9375,3)">
                <text x="21" y="15" style="color:#333333;font-size:12px;font-weight:normal;cursor:pointer;fill:#333333;" text-anchor="start">Unrented</text>
                <rect x="2" y="4" width="12" height="12" fill="#218a8c" rx="6" ry="6" class="highcharts-point"></rect>
              </g>
            </g>
          </g>
        </g>
        <g class="highcharts-axis-labels highcharts-xaxis-labels " visibility="visible">
          <text x="65.4109066825325" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:427px;text-overflow:ellipsis;" text-anchor="end" transform="translate(0,0) rotate(-45 65.4109066825325 299)" y="299" opacity="1">
            <tspan>Not Ready</tspan>
          </text>
          <text x="78.63636363636364" text-anchor="middle" y="8" transform="translate(0,0)" style="color:#666666;cursor:default;font-size:11px;fill:#666666;" visibility="visible">Studio</text>
          <text x="97.04727031889614" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:427px;text-overflow:ellipsis;" text-anchor="end" transform="translate(0,0) rotate(-45 97.04727031889614 299)" y="299" opacity="1">
            <tspan>Ready</tspan>
          </text>
          <text x="128.68363395525978" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:427px;text-overflow:ellipsis;" text-anchor="end" transform="translate(0,0) rotate(-45 128.68363395525978 299)" y="299" opacity="1">
            <tspan>Not Ready</tspan>
          </text>
          <text x="141.90909090909093" text-anchor="middle" y="8" transform="translate(0,0)" style="color:#666666;cursor:default;font-size:11px;fill:#666666;" visibility="visible">
            <tspan>1 BR</tspan>
          </text>
          <text x="160.31999759162343" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:427px;text-overflow:ellipsis;" text-anchor="end" transform="translate(0,0) rotate(-45 160.31999759162343 299)" y="299" opacity="1">
            <tspan>Ready</tspan>
          </text>
          <text x="191.95636122798706" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:427px;text-overflow:ellipsis;" text-anchor="end" transform="translate(0,0) rotate(-45 191.95636122798706 299)" y="299" opacity="1">
            <tspan>Not Ready</tspan>
          </text>
          <text x="205.1818181818182" text-anchor="middle" y="8" transform="translate(0,0)" style="color:#666666;cursor:default;font-size:11px;fill:#666666;" visibility="visible">
            <tspan>2 BR</tspan>
          </text>
          <text x="223.59272486435069" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:427px;text-overflow:ellipsis;" text-anchor="end" transform="translate(0,0) rotate(-45 223.59272486435069 299)" y="299" opacity="1">
            <tspan>Ready</tspan>
          </text>
          <text x="255.22908850071428" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:427px;text-overflow:ellipsis;" text-anchor="end" transform="translate(0,0) rotate(-45 255.22908850071428 299)" y="299" opacity="1">
            <tspan>Not Ready</tspan>
          </text>
          <text x="268.4545454545455" text-anchor="middle" y="8" transform="translate(0,0)" style="color:#666666;cursor:default;font-size:11px;fill:#666666;" visibility="visible">
            <tspan>3 BR</tspan>
          </text>
          <text x="286.86545213707797" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:427px;text-overflow:ellipsis;" text-anchor="end" transform="translate(0,0) rotate(-45 286.86545213707797 299)" y="299" opacity="1">
            <tspan>Ready</tspan>
          </text>
          <text x="318.5018157734416" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:427px;text-overflow:ellipsis;" text-anchor="end" transform="translate(0,0) rotate(-45 318.5018157734416 299)" y="299" opacity="1">
            <tspan>Ready</tspan>
          </text>
          <text x="315.90909090909093" text-anchor="middle" y="8" transform="translate(0,0)" style="color:#666666;cursor:default;font-size:11px;fill:#666666;" visibility="visible">
            <tspan>4+ BR</tspan>
          </text>
          <text x="350.1381794098052" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:427px;text-overflow:ellipsis;" text-anchor="end" transform="translate(0,0) rotate(-45 350.1381794098052 299)" y="299" opacity="1">
            <tspan>Not Ready</tspan>
          </text>
          <text x="363.3636363636364" text-anchor="middle" y="8" transform="translate(0,0)" style="color:#666666;cursor:default;font-size:11px;fill:#666666;" visibility="visible">N\A</text>
          <text x="381.77454304616884" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:427px;text-overflow:ellipsis;" text-anchor="end" transform="translate(0,0) rotate(-45 381.77454304616884 299)" y="299" opacity="1">
            <tspan>Ready</tspan>
          </text>
        </g>
        <g class="highcharts-axis-labels highcharts-yaxis-labels ">
          <text x="32" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:124px;text-overflow:clip;" text-anchor="end" transform="translate(0,0)" y="288" opacity="1">
            <tspan>0</tspan>
          </text>
          <text x="32" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:124px;text-overflow:clip;" text-anchor="end" transform="translate(0,0)" y="244" opacity="1">
            <tspan>25</tspan>
          </text>
          <text x="32" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:124px;text-overflow:clip;" text-anchor="end" transform="translate(0,0)" y="200" opacity="1">
            <tspan>50</tspan>
          </text>
          <text x="32" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:124px;text-overflow:clip;" text-anchor="end" transform="translate(0,0)" y="156" opacity="1">
            <tspan>75</tspan>
          </text>
          <text x="32" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:124px;text-overflow:clip;" text-anchor="end" transform="translate(0,0)" y="112" opacity="1">
            <tspan>100</tspan>
          </text>
          <text x="32" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:124px;text-overflow:clip;" text-anchor="end" transform="translate(0,0)" y="68" opacity="1">
            <tspan>125</tspan>
          </text>
          <text x="32" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:124px;text-overflow:clip;" text-anchor="end" transform="translate(0,0)" y="25" opacity="1">
            <tspan>150</tspan>
          </text>
        </g>
        <g class="highcharts-label highcharts-tooltip highcharts-color-undefined" style="cursor:default;pointer-events:none;white-space:nowrap;" transform="translate(64,-9999)" opacity="0" visibility="visible">
          <path fill="rgba(255, 255, 255, 0)" class="highcharts-label-box highcharts-tooltip-box" d="M 0 0 L 56 0 62 -6 68 0 125 0 C 125 0 125 0 125 0 L 125 102 C 125 102 125 102 125 102 L 0 102 C 0 102 0 102 0 102 L 0 0 C 0 0 0 0 0 0"></path>
        </g>
      </svg>
      <div class="highcharts-label highcharts-tooltip" style="position: absolute; left: 64px; top: -9999px; opacity: 0; pointer-events: none; visibility: visible;">
				<span style="font-family: &quot;Lucida Grande&quot;, &quot;Lucida Sans Unicode&quot;, Arial, Helvetica, sans-serif; font-size: 12px; position: absolute; white-space: nowrap; color: rgb(51, 51, 51); margin-left: 0px; margin-top: 0px; left: 8px; top: 8px;">
					<div class="rl-tooltip" style="border-color:#00cee6; max-width: 385px;">
						<div class="rl-tooltip-first-element">
							<span>Rented</span>
							<br>
								<span style="color:#00cee6;">48</span>
								<span style="color:#00cee6;"> / 40%</span>
							</div>
							<div class="rl-tooltip-last-element">
								<span class="rl-tooltip-smaller-text">Not Ready, 1 BR</span>
							</div>
						</div>
					</span>
      </div>
    </div>
  </div>
  <!-- end ngIf: !isEmpty -->
</div>