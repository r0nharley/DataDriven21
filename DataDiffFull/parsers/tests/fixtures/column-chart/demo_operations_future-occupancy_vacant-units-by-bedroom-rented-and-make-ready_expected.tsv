Studio:Not Ready	Rented	2.28136882129	#00cee6
Studio:Not Ready	Unrented	11.9771863118	#218a8c
Studio:Ready	Rented	2.85171102662	#00cee6
Studio:Ready	Unrented	11.9771863118	#218a8c
1 BR:Not Ready	Rented	47.9087452471	#00cee6
1 BR:Not Ready	Unrented	73.0038022814	#218a8c
1 BR:Ready	Rented	22.2433460076	#00cee6
1 BR:Ready	Unrented	46.7680608365	#218a8c
2 BR:Not Ready	Rented	67.8707224335	#00cee6
2 BR:Not Ready	Unrented	61.0266159696	#218a8c
2 BR:Ready	Rented	41.0646387833	#00cee6
2 BR:Ready	Unrented	47.9087452471	#218a8c
3 BR:Not Ready	Rented	27.9467680608	#00cee6
3 BR:Not Ready	Unrented	18.2509505703	#218a8c
3 BR:Ready	Rented	6.27376425856	#00cee6
3 BR:Ready	Unrented	10.8365019011	#218a8c
4+ BR:Ready	Rented	1.14068441065	#00cee6
4+ BR:Ready	Unrented	0	N/A
N\A:Not Ready	Rented	1.71102661597	#00cee6
N\A:Not Ready	Unrented	5.13307984791	#218a8c
N\A:Ready	Rented	0	N/A
N\A:Ready	Unrented	2.28136882129	#218a8c